website/docs/library.md

```diff
@@ -3,4 +3,4 @@ id: library
 title: Policy Library
 ---
 
-See the [Gatekeeper policy library](https://open-policy-agent.github.io/gatekeeper-library/website/) for a collection of constraint templates and sample constraints that you can use with Gatekeeper.
\ No newline at end of file
+See the [Gatekeeper policy library](https://open-policy-agent.github.io/gatekeeper-library/website/) for a collection of constraint templates, sample constraints, and sample mutation policies that you can use with Gatekeeper.
```

Comment:
do we want to updated versioned docs with this change

Reply:
done